api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -314,6 +314,12 @@ message TransformationTemplate {
   // rendered string values to be escaped in order to make valid JSON/YAML strings
   bool escape_characters = 12;
 
+  // The character to use as a delimiter when accessing dynamic metadata values
+  // in templates. Defaults to ':' if unset.
```

Comment:
why not default to dot? feels more natural?

Reply:
Because there are very often `.` in the keys, such as `envoy.filter.http` etc.

Reply:
Also, I wanted to mirror the access log formatting